silhouette/Graphtec.py

```diff
@@ -1270,6 +1266,7 @@ def plot(self, mediawidth=210.0, mediaheight=297.0, margintop=None,
     bbox['clip'] = {'urx':width, 'ury':top, 'llx':left, 'lly':height}
     bbox['only'] = bboxonly
     cmd_list = self.plot_cmds(pathlist,bbox,offset[0],offset[1])
+    print(f"Final bounding box and point counts: {bbox}", file=self.log)
```

Comment:
@gwhitney CI tests are failing probably due to this line. What happens if you drop it?

Reply:
Modified to use str() which is valid in both Python 2 and 3